config/consensus.go

```diff
@@ -811,6 +811,13 @@ func initConsensusProtocols() {
 	// FilterTimeout for period 0 should take a new optimized, configured value, need to revisit this later
 	vFuture.AgreementFilterTimeoutPeriod0 = 4 * time.Second
 
+	// Enable compact certificates.
+	vFuture.CompactCertRounds = 128
+	vFuture.CompactCertTopVoters = 1024 * 1024
+	vFuture.CompactCertVotersLookback = 16
+	vFuture.CompactCertWeightThreshold = 30
+	vFuture.CompactCertSecKQ = 128
+
```

Comment:
Do we need to initialize these values in previous consensus versions?

Reply:
Zero values are the correct initialization.